components/LeftNavbar.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+    <div class="grid">
```

Comment:
Nit: Is this grid class needed? Would like to remove it if not.

Reply:
I need to do more grid work on the page, but the main advantage is that it doesn't jump when you first load or refresh the page. For structural components, I'm wanting to use grid instead of flex.

For this one though, I think you're right and it's not doing anything yet XD